tests/test-components/pom.xml

```diff
@@ -8,12 +8,12 @@
     <parent>
         <groupId>org.mule.extensions</groupId>
         <artifactId>mule-modules-parent</artifactId>
-        <version>1.1.6</version>
+        <version>1.3.0</version>
     </parent>
 
     <groupId>org.mule.tests</groupId>
     <artifactId>test-components</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>4.4.0-SNAPSHOT</version>
```

Comment:
Isn't there some property?

Reply:
not available here, since the parent pom is not the actual parent in the module hierarchy